src/organized_segmentation/organized_segmentation_tbb.cpp

```diff
@@ -322,14 +322,15 @@ void OrganizedSegmentationTBB<PointT>::computeClusters() {
 
   clust_output_labels_ = LabelCloudPtr(new LabelCloud());
 
-  std::vector<bool> plane_labels;
-  plane_labels.resize(clust_input_label_indices_->size(), false);
+  typename pcl::EuclideanClusterComparator<PointT, pcl::Label>::ExcludeLabelSetPtr
```

Comment:
consider `boost::make_shared`

Reply:
I think this causes some issues given the type seems to be a dependent name.

```
omnimapper/src/organized_segmentation/organized_segmentation_tbb.cpp:325:30: error: dependent-name ‘pcl::EuclideanClusterComparator<PointT, pcl::Label>::ExcludeLabelSet’ is parsed as a non-type, but instantiation yields a type
  325 |   auto plane_labels = boost::make_shared<pcl::EuclideanClusterComparator<PointT, pcl::Label>::ExcludeLabelSet>();
      |                       ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```